src/main/java/core/basesyntax/strategy/BalanceOperationHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
```

Comment:
The import statement for `Storage` should be in the `db` package. Please ensure that the `Storage` class is located in the correct package.